client/influxdb.go

```diff
@@ -73,18 +74,22 @@ func ParseConnectionString(path string, ssl bool) (url.URL, error) {
 // Username/Password are optional.  They will be passed via basic auth if provided.
 // UserAgent: If not provided, will default "InfluxDBClient",
 // Timeout: If not provided, will default to 0 (no timeout)
+// InsecureSkipVerify: controls whether a client verifies the
```

Comment:
Oh, I see you're copying the config option name from the TLS client. In that case, the name must be good. :-)